components/HeroSection.vue

```diff
@@ -1,36 +1,98 @@
+              :weight="700"
+              line-height="1.2"
+            >
               Arcana Beta Testnet
             </v-heading>
-            <v-heading class="hero-section-heading-em" :level="1" :weight="700" line-height="1.2">
+            <v-heading
+              class="hero-section-heading-em"
+              :level="1"
+              :weight="700"
+              line-height="1.2"
+            >
               Quantum Ascent
             </v-heading>
           </div>
           <div>
-            <v-heading class="hero-section-heading" line-height="1.2" :weight="600">
+            <v-heading
+              class="hero-section-heading"
+              line-height="1.2"
+              :weight="600"
+              font="serif"
+            >
               Welcome!
             </v-heading>
-            <v-stack direction="column">
-              <v-text class="hero-section-launch" line-height="1.5" :weight="400">
+            <v-stack gap="0.5rem">
+              <v-text
+                class="hero-section-launch"
+                line-height="1.5"
+                :weight="500"
+              >
                 It's time to put privacy front and center.
               </v-text>
-              <v-text class="hero-section-launch" line-height="1.5" :weight="600">
+              <v-text
+                class="hero-section-launch"
+                line-height="1.5"
+                :weight="700"
+              >
                 Let's build.
               </v-text>
             </v-stack>
           </div>
         </v-stack>
-        <v-stack align="center" direction="row" sm-direction="column" justify="center" gap="2rem" sm-gap="1rem" wrap>
-          <v-button class="cta-button" :action="() => handleViewDocs()" type="button" label="View Docs"
-            label-transform="uppercase" label-size="1rem" :label-weight="600" />
-          <v-button class="cta-button" :action="() => handleGoToDashboard()" variant="outline" label="Go to Dashboard"
-            label-transform="uppercase" label-size="1rem" :label-weight="600" />
-          <a href="https://discord.gg/w6ej4FtqYS" title="Join Discord" class="cta-icon">
+        <v-stack
+          align="center"
+          direction="row"
+          sm-direction="column"
+          justify="center"
+          gap="2rem"
+          sm-gap="1rem"
+          wrap
+        >
+          <v-button
+            class="cta-button"
+            :action="() => handleViewDocs()"
+            type="button"
+            label="View Docs"
+            label-transform="uppercase"
+            label-size="1rem"
+            :label-weight="600"
+          />
+          <v-button
+            class="cta-button"
+            :action="() => handleGoToDashboard()"
+            variant="outline"
+            label="Go to Dashboard"
+            label-transform="uppercase"
+            label-size="1rem"
+            :label-weight="600"
+          />
+          <a
+            href="https://discord.gg/w6ej4FtqYS"
```

Comment:
Add a `target=_blank` to external links. Check in other places on the page as well.

Reply:
Resolved at https://github.com/arcana-network/testnet-website/pull/29/commits/bbe3968ffbd08e04c77f5113e282ee224b9b8ae3